### ISSUE TITLE
508-defect-2 [COGNITION, SCREENREADER]: Edit and update page buttons SHOULD have accessible labels on Review screen

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22339]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a screen reader user, I want to hear an accessible label like "Edit confirm eligibility" instead of "Edit" when the button has focus or virtual cursor.

## Details

All edit and update buttons do not have unique and descriptive `aria-label`s

## Acceptance Criteria

- [ ] All edit buttons have unique and descriptive `aria-label`s
- [ ] All update page buttons have unique and descriptive `aria-label`s

## Environment

* Operating System: all
* Browser: any
* Screenreading device: any
* Server destination: staging

## Steps to Recreate

1. Turn on a screen reader of your choice.
2. Enter `https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/review-and-submit` in browser.
3. Complete test case 3 and reach the review page.
4. Tab to "Edit" for contact information and issues eligible for review.
5. Verify you only hear the visible label.

## Solution 

Update the `aria-label` for each of the edit and update page buttons to reference their respective sections.

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs
<img width="1494" alt="Screen Shot 2021-03-29 at 4 12 45 PM" src="https://user-images.githubusercontent.com/14154792/112894694-2b495b80-90aa-11eb-9d78-082b24ae1d25.png">

**Comment by artsymartha68** [comment]
Interesting! It's my understanding this is another standard platform component operating OOTB. 

@cohnjesse do you happen to know if we can customize the aria labels here?

**Comment by joshkimux** [comment]
@artsymartha68 We write this ticket for almost every audit that includes a review page:

https://github.com/department-of-veterans-affairs/va.gov-team/issues/17824
https://github.com/department-of-veterans-affairs/va.gov-team/issues/17822
https://github.com/department-of-veterans-affairs/va.gov-team/issues/13014
